README.md

```diff
@@ -1,9 +1,3 @@
-# LinearAlgebra
-
-This package is part of the Julia standard library (stdlib).
-
-`LinearAlgebra.jl` provides functionality for performing linear algebra operations in Julia.
-
```

Comment:
Can you please undo this change. Eventually, there should be changes only in the two `symmetric.jl` files.